src/Tests/InstrEngineTests/InstrEngineTests/ProfilerHelpers.cs

```diff
@@ -174,9 +188,19 @@ public static void LaunchAppUnderProfiler(TestParameters parameters, string test
             }
 
             psi.EnvironmentVariables.Add(TestOutputEnvName, PathUtils.GetAssetsPath());
-            psi.EnvironmentVariables.Add(
-                is32bitTest ? HostConfig32PathEnvName : HostConfig64PathEnvName,
-                Path.Combine(PathUtils.GetAssetsPath(), string.Format(CultureInfo.InvariantCulture, "NaglerInstrumentationMethod_{0}.xml", bitnessSuffix)));
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                psi.EnvironmentVariables.Add(
+                    is32bitTest ? HostConfig32PathEnvName : HostConfig64PathEnvName,
+                    Path.Combine(PathUtils.GetAssetsPath(), string.Format(CultureInfo.InvariantCulture, "NaglerInstrumentationMethod_{0}.xml", bitnessSuffix)));
+            }
+            else // Linux
```

Comment:
> else // Linux [](http://example.com/codeflow?start=11&length=14)

We don't build for FreeBSD. We should probably condition this to Linux, and MacOS